src/sssom/cli.py

```diff
@@ -235,9 +236,10 @@ def validate(input: str, validation_types: List[SchemaValidationType]) -> None:
 @main.command()
 @input_argument
 @output_directory_option
-def split(input: str, output_directory: str) -> None:
+@click.option("--method", type=click.Choice(typing.get_args(SplitMethod)))
```

Comment:
Can we add a sensible default to `--method`? So that an execultion with `--method` does not fail because of click config?

Reply:
I think you're understanding. This PR keeps the status quo 100% but adds a new method to opt into one of two better implementations. After we merge this, then we can test against @ptgolden 's PR, and ultimately decide on the final implementation. 

WRT the --method, if none is given, it keeps the existing implementation as default.